x/auth/vesting/types/vesting_account.go

```diff
@@ -16,14 +15,14 @@ import (
 
 // Compile-time type assertions
 var (
-	_ authexported.Account        = (*BaseVestingAccount)(nil)
+	_ authtypes.AccountI          = (*BaseVestingAccount)(nil)
 	_ vestexported.VestingAccount = (*ContinuousVestingAccount)(nil)
 	_ vestexported.VestingAccount = (*PeriodicVestingAccount)(nil)
 	_ vestexported.VestingAccount = (*DelayedVestingAccount)(nil)
 )
 
 //-----------------------------------------------------------------------------
-// Base Vesting Account
+// Base Vesting AccountI
```

Comment:
nit, fix these comments

Reply:
Sure, this is unintended change from refactor